sdk/lib/core/uri.dart

```diff
@@ -50,14 +78,22 @@ const String _hexDigits = "0123456789ABCDEF";
 /// ```
 ///
 /// ## HTTP and HTTPS URI
-/// To create a URI with https scheme, use [Uri.https] or [Uri.http]:
+/// As mentioned before, these are special constructors that do the same thing
```

Comment:
I don't think they have been mentioned before. Also it's not clear which *constructors* are referenced here, since they haven't been mentioned before either.

Something like:
```dart
/// The [Uri.http] and [Uri.https] constructors are specialized for creating
/// URIs with the `http:` and `https:` schemes, and doing so more conveniently
/// than through the general [new Uri] constructor.